v2/googlecloud-to-elasticsearch/src/main/resources/bigquery-to-elasticsearch-command-spec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "mainClass": "com.google.cloud.teleport.v2.elasticsearch.templates.BigQueryToElasticsearch",
+  "classPath": "/template/googlecloud-to-elasticsearch/*:/template/googlecloud-to-elasticsearch/libs/*:/template/googlecloud-to-elasticsearch/classes",
```

Comment:
classpath should be bigquery-to-elasticsearch instead of googlecloud-to-elasticsearch
e.g:  /template/bigquery-to-elasticsearch/